README.md

```diff
@@ -21,12 +21,19 @@ To start using the sniffer, you must flash the firmware, install the script, and
 nrfjprog -f nrf52 --program nrf802154_sniffer/nrf802154_sniffer.hex --chiperase -r
 ```
 
-### Install extcap script
+### Install extcap script on Linux
```

Comment:
I would keep `### Install extcap script` and add `#### Linux` and `#### Windows` as subsections.